src/Console/GeneratorCommand.php

```diff
@@ -50,17 +49,14 @@ class GeneratorCommand extends Command
 
     protected $onlyExtend;
 
-
     /**
-     *
      * @param \Illuminate\Config\Repository $config
      * @param \Illuminate\Filesystem\Filesystem $files
      * @param \Illuminate\View\Factory $view
      */
     public function __construct(
-        /*ConfigRepository */ $config,
+        $config,
```

Comment:
I actually removed them, they cause discrepancy between running php-cs-fixer locally vs. on github.

Should not happen but TBH this is so unorthodox I simply removed it; the phpdoc above is correct(er) anyway.